cairo/ethereum/cancun/fork_types.cairo

```diff
@@ -1,12 +1,36 @@
 from starkware.cairo.common.alloc import alloc
 
-from ethereum_types.bytes import Bytes20, Bytes256, Bytes, BytesStruct
-from ethereum_types.numeric import Uint, U256, U256Struct
+from ethereum_types.bytes import Bytes20, Bytes32, Bytes256, Bytes, BytesStruct
+from ethereum_types.numeric import Uint, U256, U256Struct, bool
 from ethereum.crypto.hash import Hash32
 
 using Address = Bytes20;
 using Root = Hash32;
 
+struct TupleAddressBytes32 {
```

Comment:
Nit: It is implicitly agreed to define first the more internal type first so 
first TupleAddressBytes32Struct then TupleAddressBytes32
Same way like you did for SetTupleAddressBytes32